### DISCUSSION TITLE
Cannot install Module with MacOS

### DISCUSSION THREAD

**Discussion by kung1388** [UnsignedArduino/CircuitPython-Bundle-Manager-v2#9] [Q&A]
Following Install from source for MacOS. It cannot install Module until create "boot_out.txt" text file and "lib" folder in Selected device.

**Comment by UnsignedArduino** [comment]
Oh yea, you need a `boot_out.txt` file on a drive for it to be recognized as a CircuitPython device. 
For the `lib` folder, that should be automatically created, but I don't think I have any checks for that - I'll open an issue for that

**Reply by UnsignedArduino** [reply]
Here: (for completeness) https://github.com/UnsignedArduino/CircuitPython-Bundle-Manager-v2/issues/10
